scipy/stats/_continuous_distns.py

```diff
@@ -846,8 +846,28 @@ def asymptotic_ab_large(a, b):
             t3 = -50/b - 10*b**-2.0 - b**-3.0 + b**-4.0
             return log_term + (t1 + t2 + t3) / 120
 
+        def asymptotic_b_large(a, b):
+            sum_ab = a + b
+            t1 = sc.gammaln(a) - (a - 1) * sc.psi(a)
+            t2 = (
+                - 1/(2*b) + 1/(12*b) - b ** -2.0/ 12 - 1/(12*sum_ab)
+                + 1/sum_ab + sum_ab**-2.0/6
```

Comment:
I would have expected also terms with the powers -4 and -6 here as the expansion is:
![image](https://github.com/scipy/scipy/assets/40656107/7eceb1ee-7fe0-4ef1-a589-563899e2aca8)

Reply:
I think for the last term in this expansion we should have - 1/(252*x**6) because simply 1/252**6 will cause huge values when this is multiplied by a large value of b e.g. 1e50.